moban/mobanfile/targets.py

```diff
@@ -34,6 +39,25 @@ def _handle_explicit_target(options, target):
     data_file = target.get(constants.LABEL_CONFIG, common_data_file)
     output = target[constants.LABEL_OUTPUT]
     template_type = target.get(constants.LABEL_TEMPLATE_TYPE)
+    if template_type and len(template_type) > 0:
+        if constants.TEMPLATE_TYPES_FILE_EXTENSIONS in template_type:
+            reporter.report_file_extension_not_needed()
```

Comment:
@chfw How would I test this line? I tried using `contextlib.redirect_stdout`, but it doesn't seem to work.